### DISCUSSION TITLE
Escaping special characters

### DISCUSSION THREAD

**Discussion by vincentpace** [tone-row/flowchart-fun#147] [Q&A]
Is there a way to escape special characters, say, if you want to use a colon ":" or parentheses "()" as text in your boxes? I tried backslashes but that didn't work.

**Comment by rob-gordon** [comment]
There is. Text needs to be uri-encoded (here's a [helpful tool](https://meyerweb.com/eric/tools/dencoder/) to do this online). 

See https://github.com/tone-row/flowchart-fun/issues/72#issuecomment-818830859 for more info

**Reply by rob-gordon** [reply]
@vincentpace let me know if that doesn't solve your particular case. Thanks!

**Reply by vincentpace** [reply]
@rob-gordon Thanks, that did the trick!
